### DISCUSSION TITLE
Question about the function "get_results_for_members_internal_forces_by_member_set"

### DISCUSSION THREAD

**Discussion by peu-oliveira** [Dlubal-Software/Dlubal_CSharp_Client#61] [Q&A] [answered]
I am using the RSTAB9 API and when I use the function "get_results_for_members_internal_forces_by_member_set" it requires the following input: "(case_object_types loading type, int loading_no, object_location[] object locations, axes_type member_result_coordinate_system_type)". I am not being able to identify the object locations of my member set , is there a solution to it?

**Comment by Rebecca-Coding21** [comment, marked as answer]
Hi @peu-oliveira ,
you can create an object_location[] as shown in the following example:

`object_location[] memberSets= new object_location[1];
 
object_location object_loc = new object_location{
    type = object_types.E_OBJECT_TYPE_MEMBER,
    no = 3,
    parent_no = 0
};
memberSets[0] = object_loc;`

You can also find out about the structure of object_location by using the tools Boomerang or SoapUI. The usage of both tools is described in the [article ](https://dlubal-software.github.io/.github/guide/net/netArticle.html#verification-of-webservice-operations-with-boomerang-and-soapui) in our documentation.
